source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/.eslintignore

```diff
@@ -0,0 +1,5 @@
+lib/*.js
+test/*.js
+*.d.ts
+coverage
+test/lambda/index.js
```

Comment:
I think we can drop this line in this construct

Reply:
Can't delete as it raises some parsing errors